pgsmo/objects/table_objects/trigger.py

```diff
@@ -165,7 +161,7 @@ def _delete_query_data(self) -> dict:
                 "nspname": self.parent.schema,
                 "relname": self.parent.name
             },
-            "cascade": self.cascade
+            "cascade": True
```

Comment:
Will this allow trigger functions to be seen under trigger?

Reply:
No, this simply adds `CASCADE` to the drop script!